### PR TITLE
fix(Metrics Router): add new regions br-sao, ca-tor, jp-osa and jp-tok

### PR DESCRIPTION
**Pull request by PhaniDivi-613**
## PR summary
Added new regions Sao Paulo(br-sao), Toronto(ca-tor), Osaka(jp-tok) and Tokyo(jp-tok) to metrics-router

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## Current vs new behavior  
Added new region. No change in old code

## Does this PR introduce a breaking change?    
- [ ] Yes
- [x] No

## Other information

### CODE REVIEW

File: metricsrouterv3/metrics_router_v3.go [hunk 1]
```diff
@@ -114,6 +114,10 @@ func GetServiceURLForRegion(region string) (string, error) {
 	var endpoints = map[string]string{
 		"au-syd":           "https://au-syd.metrics-router.cloud.ibm.com/api/v3",           // The server for IBM Cloud Metrics Routing Service in the au-syd region.
 		"private.au-syd":   "https://private.au-syd.metrics-router.cloud.ibm.com/api/v3",   // The server for IBM Cloud Metrics Routing Service in the au-syd region.
+		"br-sao":           "https://br-sao.metrics-router.cloud.ibm.com/api/v3",           // The server for IBM Cloud Metrics Routing Service in the br-sao region.
```
Comment (padamstx) [inline metricsrouterv3/metrics_router_v3.go:117]: Did you manually update the generated code, or did you re-generate the code with the SDK generator and an updated API definition?
If the former, then please re-do your PR by re-generating the code using the updated API definition.

Comment (PhaniDivi-613) [reply metricsrouterv3/metrics_router_v3.go:117]: No I have generated using SDK generator

**Review by padamstx — APPROVED** [review]
LGTM

**Pull request merged**